reconcile/templating/renderer.py

```diff
@@ -256,10 +272,17 @@ def run(self, dry_run: bool) -> None:
                 vcs=vcs,
                 parser=create_parser(),
             )
-            persistence = GitlabFilePersistence(vcs, merge_request_manager)
+            url = get_app_interface_repo_url()
 
-            state = init_state(QONTRACT_INTEGRATION, self.secret_reader)
+            with tempfile.TemporaryDirectory(
+                prefix=join_path(self.params.git_clone_path, "clone"),
```

Comment:
`prefix` doesn't change the destination directory, it'll be still `$TMP`. The correct parameter is `dir`

Reply:
since we only need one tmp dir, can use integration name as prefix, like `QONTRACT_INTEGRATION`